### ISSUE TITLE
Half logged-in state has no "Log out" button

### ISSUE THREAD

**Issue by paulproteus** [sandstorm-io/sandstorm#970]
Steps to reproduce:

* Visit oasis.sandstorm.io

* Click "Sign in" and choose (e.g.) Google

* Look at the profile prompt and decide you don't care and just want to sign out

* Look for a "Log out" link in the top-right

Expected behavior:

* There is a "Log out" link in the top-right

Actual behavior:

* There is no "Log out" link in the top-right

Proposed fix:

* When the user is creating their profile, we should show a "Log out" link in the top right.

Feedback requested:

* I'd love a sanity-check on the above from e.g. @neynah and @dwrensha (others' sanity-checks are useful too)

**Comment by neynah** [comment]
Yes! I actually noticed this as well when I was setting up a demo acccount. I agree that there should be a log-out button or at least a "cancel" button during the "confirm your profile" process.
